### PR TITLE
[EASI-2575] Deploy Faktory with GitHub Actions

### PR DESCRIPTION
**Pull request by ClayBenson94**
# EASI-2575

## Changes and Description

- Modify Build Images pipeline to push a Faktory image to ECR
- Modify Deploy to Environment pipeline to deploy Faktory as an ECS service

## How to test this change

- Monitor the CI run that happens in this PR!
  - Like [this one here](https://github.com/CMSgov/mint-app/actions/runs/3914282961)!

## PR Author Review Checklist

- [ ] Met the ticket's acceptance criteria, or will meet them in a subsequent PR.
- [ ] Added or updated tests for backend resolvers or other functions as needed.
- [ ] Added or updated client tests for new components, parent components, functions, or e2e tests as necessary.
- [ ] Updated the [Postman Collection](../MINT.postman_collection.json) if necessary.

## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.

### CODE REVIEW

**Comment by ClayBenson94** [comment]
We don't want to merge this yet (Faktory isn't in any of our envs but `dev`) but nothing about this PR should change when we deploy to other envs!

File: .github/workflows/deploy_pipeline.yml [hunk 1]
```diff
@@ -42,6 +44,7 @@ jobs:
       ECR_REGISTRY: ${{ secrets.ECR_REGISTRY }}
 
   deploy_test:
+    if: ${{ false }}
```
Comment (ClayBenson94) [inline .github/workflows/deploy_pipeline.yml:47]: TODO: Remove this!!

**Review by StevenWadeOddball — DISMISSED** [review]
This looks logical to me! Approving on virtue of successful run in Github Actions

**Review by Jdwoodson — APPROVED** [review]
LGTM

**Pull request merged**